board/safety/safety_honda.h

```diff
@@ -441,9 +441,9 @@ static int honda_bosch_fwd_hook(int bus_num, int addr) {
     bus_fwd = 2;
   }
   if (bus_num == 2)  {
-    int is_lkas_msg = (addr == 0xE4) || (addr == 0xE5) || (addr == 0x33D) || (addr == 0x33DA) || (addr == 0x33DB);
-    int is_acc_msg = ((addr == 0x1C8) || (addr == 0x30C)) && honda_bosch_radarless && honda_bosch_long;
-    bool block_msg = is_lkas_msg || is_acc_msg;
+    const bool is_lkas_msg = (addr == 0xE4) || (addr == 0xE5) || (addr == 0x33D) || (addr == 0x33DA) || (addr == 0x33DB);
+    const bool is_acc_msg = ((addr == 0x1C8) || (addr == 0x30C)) && honda_bosch_radarless && honda_bosch_long;
+    const bool block_msg = is_lkas_msg || is_acc_msg;
```

Comment:
`honda_nidec_fwd_hook` was already forgotten to be updated. going to revert the const changes, could have been intentional before